cmd/operator/deploy/operator/05-deployment.yaml

```diff
@@ -43,6 +43,7 @@ spec:
       - name: operator
         image: gke.gcr.io/prometheus-engine/operator:v0.4.1-gke.0
         args:
+        - "--operator-namespace=gmp-system"
         - "--public-namespace=gmp-public"
         - "--priority-class=gmp-critical"
         - "--image-collector=gke.gcr.io/prometheus-engine/prometheus:v2.28.1-gmp.7-gke.0"
```

Comment:
I guess these flags are still valid until we publish a new image version.

Bump the image-collector to latest?

Reply:
Yea, but of temporary  drift. For that version of the operator our canonical version didn't change yet basically. So I'd say this just gets updated with the v0.5.0 release?

Reply:
Ah makes sense.